ParquetClassLibrary/Sandbox/MapRegion.cs

```diff
@@ -399,27 +397,28 @@ public ParquetStack GetAllParquetsAtPosition(Vector2Int in_position)
                                     GetCollectableAtPosition(in_position));
         }
 
-
         /// <summary>
         /// Gets all the parquets in the entire region.
         /// </summary>
         /// <returns>A collection of parquets.</returns>
         public IEnumerable<ParquetParent> GetAllParquets()
         {
+            // TODO: This is not optimal — it'd be nice to not generate these garbage objects full of duplicates
+            // Is there a way we can just use IDs?
```

Comment:
Given the most common use of this function, if we passed back IDs we'd end up immediately asking what all the IDs are. Seems fine!

We had an offline chat about this, and it seemed favorable to do it the way you have it here.